pkg/apis/kubermatic/v1/audit_logging.go

```diff
@@ -53,4 +53,18 @@ type AuditLoggingSettings struct {
 	PolicyPreset AuditPolicyPreset `json:"policyPreset,omitempty"`
 	// Optional: Configures the fluent-bit sidecar deployed alongside kube-apiserver.
 	SidecarSettings *AuditSidecarSettings `json:"sidecar,omitempty"`
+	// Optional: Configures the fluent-bit sidecar deployed alongside kube-apiserver.
```

Comment:
Is the comment actually correct? And if so, why do we have 2 settings to "configure the fluent-bit sidebar" (`sidecar` & `webhookBackend`)?